modules/game/client/directives/drawing.client.directives.js

```diff
@@ -162,12 +166,26 @@ angular.module('game').directive('dtDrawing', ['Socket', 'MouseConstants', 'Canv
             lastX = mouse.x;
             lastY = mouse.y;
           }
+        }, {passive: false});
```

Comment:
what does this change do?

Reply:
It was required in the touch event listeners to deal with [https://github.com/inuyaksa/jquery.nicescroll/issues/799](url) but it looks like it can actually be taken out of the mouse event event listeners